src/main/java/com/manardenza/utils/DatabaseSeeder.java

```diff
@@ -1,12 +1,15 @@
 package com.manardenza.utils;
 
 import com.manardenza.dao.HotelDaoImpl;
+import com.manardenza.dao.ReservationDaoImpl;
 import com.manardenza.dao.UserDaoImpl;
 import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Reservation;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.manardenza.entity.Reservation'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)